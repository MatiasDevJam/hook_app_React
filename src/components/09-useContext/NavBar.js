import React from 'react';
import { NavLink } from 'react-router-dom'

export const NavBar = () => {
    return (
        <nav className="nav">
            
            <NavLink exact activeClassName="active" to="/" className="nav-link" >Home</NavLink>
            <NavLink exact activeClassName="active" to="/login" className="nav-link" >Login</NavLink>
            <NavLink exact activeClassName="active" to="/about" className="nav-link" >About</NavLink>
            
        </nav>
    )
}
