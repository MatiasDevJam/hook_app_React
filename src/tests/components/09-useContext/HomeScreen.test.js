import React from 'react';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Prueba en <HomeScreen />', () => {

    const user = {
        name: 'Matias',
        email: 'mail@mail.com'
    }

    const wrapper = mount( 
        <UserContext.Provider value={{
            user
        }}>
            <HomeScreen />
        </UserContext.Provider>
    );

    test('debe mostrarse correctamente', () => {

        expect( wrapper ).toMatchSnapshot();
    })
    
    
})

