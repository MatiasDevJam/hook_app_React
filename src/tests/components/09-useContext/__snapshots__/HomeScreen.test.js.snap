// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <HomeScreen /> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Matias",
   "email": "mail@mail.com"
}
  </pre>
</div>
`;
